src/components/Timer/Timer.js

```diff
@@ -1,40 +1,47 @@
 import React, { useState, useEffect } from 'react';
 
 import styled from 'styled-components';
-import {post} from '../../requests';
+import { post } from '../../requests';
 
 const Timer = () => {
-    const SPACE = 32
-    const [time, setTime] = useState("00:00:00")
+    const SPACE = 32;
+    const countdownInMills = 15000;
```

Comment:
Let's focus on the purpose of this variable and rename it to `maxCountdown` or `maxCountownTime`.